Cargo.toml

```diff
@@ -195,11 +195,11 @@ openssl = "0.10.72"
 pairing = "0.23.0"
 parking_lot = "0.12"
 pbkdf2 = { version = "0.11.0", default-features = false }
-proc-macro2 = "1.0.93"
+proc-macro2 = "1.0.101"
 proptest = "1.6"
 qstring = "0.7.2"
 qualifier_attr = { version = "0.2.2", default-features = false }
-quote = "1.0.35"
+quote = "1.0.41"
 rand = "0.8.5"
```

Comment:
Looks like one of these dependencies is causing `rand` to be resolved to `0.9.x` in the `Cargo.lock` file. Do you mind reviewing the lock file changes to ensure they're all ok? I think the 0.9 upgrade is fine for `rand` since we're [working on upgrading `rand` in agave](https://github.com/anza-xyz/agave/issues/4738) to that version as well. But it should probably be reflected here in the `Cargo.toml` too if it's the new min?